From 9e00e702633c47c590a869bc66b5c2ceec09da50 Mon Sep 17 00:00:00 2001
From: Dominik Röttsches <drott@chromium.org>
Date: Tue, 28 Sep 2021 15:31:50 +0000
Subject: [PATCH] Roll src/third_party/harfbuzz-ng/src/ 6602cbb70..a52c6df38 (80 commits)

Fixed: 1252284
Change-Id: Ie2ff99da5e41ca97b8881e1bd3e158881f4d5a8e
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3190252
Commit-Queue: Dominik Röttsches <drott@chromium.org>
Commit-Queue: Calder Kitagawa <ckitagawa@chromium.org>
Auto-Submit: Dominik Röttsches <drott@chromium.org>
Reviewed-by: Calder Kitagawa <ckitagawa@chromium.org>
Reviewed-by: Anders Hartvoll Ruud <andruud@chromium.org>
Cr-Commit-Position: refs/heads/main@{#925776}

(only components/paint_preview/common/subset_font.cc)
---

diff --git a/components/paint_preview/common/subset_font.cc b/components/paint_preview/common/subset_font.cc
index 45daea7b1..43a448e 100644
--- a/components/paint_preview/common/subset_font.cc
+++ b/components/paint_preview/common/subset_font.cc
@@ -79,7 +79,7 @@
   // Retain all variation information for OpenType variation fonts. See:
   // https://docs.microsoft.com/en-us/typography/opentype/spec/otvaroverview
   hb_set_t* skip_subset =
-      hb_subset_input_no_subset_tables_set(input.get());  // Owned by |input|.
+      hb_subset_input_set(input.get(), HB_SUBSET_SETS_NO_SUBSET_TABLE_TAG);
   hb_set_add(skip_subset, HB_TAG('a', 'v', 'a', 'r'));
   hb_set_add(skip_subset, HB_TAG('c', 'v', 'a', 'r'));
   hb_set_add(skip_subset, HB_TAG('f', 'v', 'a', 'r'));
